view/rooms/index.pug

```diff
@@ -11,7 +11,7 @@ html
     title Refocus Room
     link(rel='shortcut icon', href='favicon.ico')
     link(rel='stylesheet', type='text/css', href='../static/css/salesforce-lightning-design-system.min.css')
-    script(src='http://code.jquery.com/jquery-1.11.1.js')
+    script(src='https://code.jquery.com/jquery-1.11.1.js')
```

Comment:
if loading remote resources (and why loading remote resources?) should loading the minified versions of them